django_project/fish/models/iucn_status.py

```diff
@@ -4,29 +4,50 @@
 """
 
 from django.db import models
+from django.dispatch import receiver
 
 
 class IUCNStatus(models.Model):
     """IUCN status model."""
+    CATEGORY_CHOICES = (
+        ('LC', 'Least Concern'),
+        ('NT', 'Near Threatened'),
+        ('VU', 'Vulnerable'),
+        ('EN', 'Endangered'),
+        ('CR', 'Critically Endangered'),
+        ('EW', 'Extinct In The Wild'),
+        ('EX', 'Extinct'),
+    )
 
-    name = models.CharField(
-        max_length=100,
+    category = models.CharField(
+        max_length=50,
+        choices=CATEGORY_CHOICES,
         blank=True,
         default='',
     )
-    narrative = models.TextField(
-        blank=True,
-    )
     sensitive = models.BooleanField(
         default=False
     )
 
     def __str__(self):
-        return u'%s' % self.name
+        return u'%s' % self.category
 
     # noinspection PyClassicStyleClass
     class Meta:
         """Meta class for project."""
         app_label = 'fish'
         verbose_name_plural = 'IUCN Status'
         verbose_name = 'IUCN Status'
+
+
+@receiver(models.signals.pre_save, sender=IUCNStatus)
+def iucn_status_pre_save_handler(sender, instance, **kwargs):
+    if instance.category:
+        # if the category is Critically Endangered or Endangered or
+        # Vulnerable then the iucn status is sensitive
+        if instance.category == 'CR' or \
```

Comment:
maybe we  make list for this on model class?
SENSITIVE_STATUS = ['CR','EN','VU']
or maybe giving sensitive status as third element on CATEGORY_CHOICES